### PR TITLE
module sharepoint

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

**Comment by cbourgois** [comment]
We have to transform translations `xml` to `json`.

File: packages/manager/modules/sharepoint/.htmlhintrc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "inline-style-disabled": false,
+  "space-tab-mixed-disabled": "space",
+  "spec-char-escape": true,
+  "src-not-empty": true,
+  "style-disabled": true,
+  "tag-pair": true,
+  "tag-self-close": false,
+  "tagname-lowercase": true,
+  "title-require": true
+}
```
Comment (marie-j) [inline packages/manager/modules/sharepoint/.htmlhintrc]: Isn't there a shared configuration for this one ?

**Comment by jleveugle** [comment]
> We have to transform translations `xml` to `json`.

done

File: packages/manager/modules/sharepoint/.htmlhintrc (hunk 1 above)
Comment (jleveugle) [reply packages/manager/modules/sharepoint/.htmlhintrc]: Not in this case, it's a specific one for that module (options are different from the root file :/)

**Pull request merged**